Makefile

```diff
@@ -13,13 +13,16 @@ build: ## Build all Docker images
 	@docker build . --file Dockerfile --tag ${IMAGE_NAME}:latest
 
 up: ## Start all Docker services
-	@docker-compose up -d
+	@docker-compose up --remove-orphans  -d
 
 stop: ## Stop all Docker services
 	@docker-compose stop
 
 down: ## Down all Docker services
-	@docker-compose down
+	@docker-compose down -v
+
+clean: ## Clean all Docker services
+	@docker rmi -f lumen-api-starter-app
```

Comment:
Removing `volumes` with `-v` is risky. Developer may not want to delete the volumes when they want to remove only the instances.

I will suggest the following instead -

```
down: ## Down all Docker services
 	@docker-compose down

remove: ## Remove all Docker instances, images, volumes and networks
    @docker-compose down -v
 	@docker rmi -f lumen-api-starter-app
```